src/server.ts

```diff
@@ -82,14 +76,12 @@ export class AttendingServer {
             }
         }
         if (queue === undefined) {
-            return `There is not a queue with the name ${queue_name}`
-            //throw new UserError(`There is not a queue with the name ${queue_name}`)
+            throw `There is not a queue with the name ${queue_name}`
```

Comment:
As per my comment above, this should also probably be a UserError. IIRC throwing strings is generally frowned upon in JS best practices.